### ISSUE TITLE
FR create objects with a specific tag set

### ISSUE THREAD

**Issue by gurubert** [minitriga/Netbox-Device-Type-Library-Import#104]
I would like to see the possibility to specify a NetBox tag on the command line that is then set on every object that gets created.
This way the imported device types etc are easily told apart from other device types etc.

**Comment by danner26** [comment]
Are you saying that you'd like a switch to tag all imported devices/modules? For instance with `imported-devicetype-library` as an example?

**Comment by gurubert** [comment]
Yes, exactly.

**Comment by danner26** [comment]
Makes sense to me! We would need to create the tag first, and then utilize it.. I will have to look into this but if you want to take a crack feel free to
